python/MooseDocs/extensions/media.py

```diff
@@ -172,6 +173,7 @@ def defaultSettings():
         settings['tstart'] = (None, "Time (sec) to start video.")
         settings['tstop'] = (None, "Time (sec) to stop video.")
         settings['poster'] = (None, "Add a 'poster' image the the video")
+        settings['quicktime'] = (None, "Video to utilize Macintosh codecs (for alpha transparencies)")
```

Comment:
Using the `video/quicktime` tag type doesn't cause an issue on non-Mac systems?

Reply:
nope nope... this is "the way" according to the masses. Ref: https://www.w3schools.com/html/html5_video.asp

Basically the web browser will continue down the list till it find a codec it likes...

Reply:
Nifty!